pkg/cmd/openshift-tests/run-test/command.go

```diff
@@ -78,5 +79,6 @@ func NewRunTestCommand(streams genericclioptions.IOStreams) *cobra.Command {
 	cmd.Flags().StringSliceVar(&testOpt.ExactMonitorTests, "monitor", testOpt.ExactMonitorTests,
 		fmt.Sprintf("list of exactly which monitors to enable. All others will be disabled.  Current monitors are: [%s]", strings.Join(monitorNames, ", ")))
 	cmd.Flags().StringSliceVar(&testOpt.DisableMonitorTests, "disable-monitor", testOpt.DisableMonitorTests, "list of monitors to disable.  Defaults for others will be honored.")
+	cmd.Flags().BoolVar(&testOpt.RemoveMonitor, "remove-monitor", true, "remove all monitors. Defaults to false.")
```

Comment:
Monitor tests are the most valuable tests we have.  Why are they being made mass-removable here?  IPSec must be able to pass the monitor tests or have an explanation why IPSec is incompatible with a particular one and update the test to understand that.